packages/react/tooltip/src/Tooltip.tsx

```diff
@@ -125,44 +133,50 @@ interface TooltipProps {
   open?: boolean;
   defaultOpen?: boolean;
   onOpenChange?: (open: boolean) => void;
-
   /**
    * The duration from when the pointer enters the trigger until the tooltip gets opened. This will
    * override the prop with the same name passed to Provider.
    * @defaultValue 700
    */
   delayDuration?: number;
+  /**
+   * Whether to close the tooltip immediately when the pointer leaves the trigger.
+   * @defaultValue false
+   */
+  disableHoverableContent?: boolean;
 }
 
 const Tooltip: React.FC<TooltipProps> = (props: ScopedProps<TooltipProps>) => {
+  const providerContext = useTooltipProviderContext(TOOLTIP_NAME, props.__scopeTooltip);
```

Comment:
I was previously using it to assign a default value in the props destructure (as we do with the portal force mount stuff), it can be moved back now though.